nixos/modules/system/boot/stage-2-init.sh

```diff
@@ -167,10 +167,6 @@ exec 1>&$logOutFd 2>&$logErrFd
 exec {logOutFd}>&- {logErrFd}>&-
 
 
-# Start systemd.
+# Start systemd in a clean environment.
```

Comment:
What if somebody was setting some environment variable in `postBootCommands` and expecting that to be there for systemd? Is that a conceivable use case?

Reply:
I doubt it since systemd doesn't propagate these variables to processes afaik. I could add an option for additional `ManagerEnvironment=` variables if you like

Reply:
Hm. Maybe that would be good, yes. Even if it doesn't get passed on, you know the kinds of setups people have and the features they (ab)use.

Looking at some of [the environment variables systemd apparently understands](https://www.freedesktop.org/software/systemd/man/systemd.html#Environment), I'm sure there's _someone_ out that sets, like… `SYSTEMD_LOG_COLOR`, for _some_ reason.

Reply:
Like this?

Reply:
probably, yes